docs/topics/clusterdefinitions.md

```diff
@@ -235,6 +235,30 @@ Additionally above, we specified a custom docker image for tiller, let's say we
 
 The reason for the unsightly base64-encoded input type is to optimize delivery payload, and to squash a human-maintainable yaml file representation into something that can be tightly pasted into a JSON string value without the arguably more unsightly carriage returns / whitespace that would be delivered with a literal copy/paste of a Kubernetes manifest.
 
+#### coredns
```

Comment:
Should we refer to it consistently as `CoreDNS` as the project would prefer? Except when referring to the AKS Engine plugin specifically.